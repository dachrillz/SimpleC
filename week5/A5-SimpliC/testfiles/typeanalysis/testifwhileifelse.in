int main(){

    while(1){
        return 0;
    }
    
    if(2){
        return 0;
    }
    
    if(2){
        return 0;
    }
    else{
        return 0;
    }

}
