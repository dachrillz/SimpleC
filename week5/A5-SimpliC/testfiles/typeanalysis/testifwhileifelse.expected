Error at line 3: Must have boolean expression in While. Got: int
Error at line 7: Must have boolean expression in If. Got: int
Error at line 11: Must have boolean expression in IfElse. Got: int
